apps/ledger-live-desktop/.flowconfig

```diff
@@ -26,10 +24,14 @@ module.name_mapper='^@ledgerhq/live-common' ->'<PROJECT_ROOT>/../../libs/ledger-
 module.name_mapper='^@ledgerhq/live-common/\(.*\)' ->'<PROJECT_ROOT>/../../libs/ledger-live-common/lib/\1'
 munge_underscores=true
 esproposal.optional_chaining=enable
+module.file_ext=.ts
```

Comment:
Do we want flow check on ts files ?

Reply:
Would say no

Reply:
What's the reasoning behind adding this here @sprohaszka-ledger?